test/kvao/delete-all.suite.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const bdd = require('../helpers/bdd-if');
+const helpers = require('./_helpers');
+const should = require('should');
+
+module.exports = function(dataSourceFactory, connectorCapabilities) {
+  var supportsDeleteAllOperation =
+    connectorCapabilities.supportsDeleteAllOperation !== false;
```

Comment:
As I commented earlier in another PR, the test suite should skip new features that are not supported by the connector-under-test yet. That way downstream builds stay green and we can land a new feature before it's implemented in connectors.

```js
var supportsDeleteAllOperation =  !!connectorCapabilities.supportsDeleteAllOperation;
```

I would also consider using a shorter flag name, e.g. `canDeleteAll`, `hasDeleteAll` or `providesDeleteAll`. In fact, can we check the `connector` object to detect whether it implements deleteAll or not?

```js
var supportsDeleteAll = 'deleteAll' in dataSourceFactory().connector;
```

That way there is no need for a flag in `connectorCapabilities`.